models/Document/Dao.php

```diff
@@ -98,7 +98,7 @@ public function update()
     {
         $typeSpecificTable = null;
         $validColumnsTypeSpecific = [];
-        if (in_array($this->model->getType(), ['email', 'newsletter', 'hardlink', 'link', 'page', 'snippet'])) {
+        if (in_array($this->model->getType(), ['email', 'newsletter', 'hardlink', 'link', 'page', 'snippet', 'headlesspage'])) {
```

Comment:
I guess it's time to move the available types to Symfony config, wdyt?

Reply:
yes, makes sense! Moved it 😉